apps/google-analytics-4/lambda/src/lib/errors/middlewares.ts

```diff
@@ -11,7 +11,7 @@ export type ApiErrorMap = Record<string, (e: any) => ApiError<Record<string, unk
 
 export const apiErrorHandler: ErrorRequestHandler = (error, _request, response, next) => {
   if (error) {
-    console.log(JSON.stringify(error));
+    console.error(JSON.stringify(error));
```

Comment:
We need a proper logger! 🙈  (but not this very second)

Reply:
Yeah, what are other folks using? I'd love to see us adopt Datadog for *everything* (logging, metrics, tracing, RUM, etc)